rflx/generator/session.py

```diff
@@ -0,0 +1,2796 @@
+# pylint: disable = too-many-lines
+
+from dataclasses import dataclass, field as dataclass_field
+from typing import Callable, Iterable, List, NoReturn, Sequence, Set, Tuple
+
+from rflx import (
+    declaration as decl,
+    expression as expr,
+    identifier as rid,
+    model,
+    statement as stmt,
+    typing_ as rty,
+)
+from rflx.ada import (
```

Comment:
The current approach is consistent to all other modules in `rflx/generator` except `common.py`. I'm not sure if we should generally use the `ada.` prefix in the generator modules. I have the feeling that this might decrease the readability. What do you think?

Reply:
Let's leave it as is then.